src/main/java/com/redhat/devtools/intellij/tektoncd/ui/inspector/VariableReferencesInspector.java

```diff
@@ -8,10 +8,9 @@
  * Contributors:
  * Red Hat, Inc.
  ******************************************************************************/
-package com.redhat.devtools.intellij.tektoncd.utils;
+package com.redhat.devtools.intellij.tektoncd.ui.inspector;
```

Comment:
An inspector is a background task and is not part of ui in an MVC word so would be better moved to another package